frontend/packages/pipelines-plugin/src/components/pipelines/utils/pipeline-operator.ts

```diff
@@ -44,3 +44,8 @@ export const isGAVersionInstalled = (operator: SemVer): boolean => {
   if (!operator) return false;
   return gte(operator.version, PIPELINE_GA_VERSION);
 };
+
+export const isTriggersGAVersionInstalled = (operator: SemVer): boolean => {
```

Comment:
nit: name seems to be little long, can we simply keep it to `isTriggersGAVersion`